pkg/controllers/configobservation/oauth/idp_conversions.go

```diff
@@ -425,6 +425,10 @@ func checkOIDCPasswordGrantFlow(
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode >= 500 && resp.StatusCode < 600 {
```

Comment:
nit: 600 reads weird in http :sweat_smile: Would you mind using `<=599` instead?

Reply:
Sure thing 😄